r/adbcdrivermanager/configure

```diff
@@ -25,28 +25,28 @@ if [ -f bootstrap.R ]; then
   $R_HOME/bin/Rscript bootstrap.R --vanilla
 fi
 
-if [ -f "src/adbc.h" ] && [ -f "src/adbc_driver_manager.h" ] && [ -f "src/adbc_driver_manager.cc" ]; then
+if [ -f "src/arrow-adbc/adbc.h" ] && [ -f "src/arrow-adbc/adbc_driver_manager.h" ] && [ -f "src/adbc_driver_manager.cc" ]; then
   echo "Found vendored ADBC"
   exit 0
 fi
 
 # We have a situation where the package has been built via R CMD build
 # manually or with pkgbuild <=1.4.0. This is common (e.g., pak::pkg_install()
 # or remotes::install_github() or any dependency setups that use either method)
-curl -L https://github.com/apache/arrow-adbc/raw/main/adbc.h \
-  --output src/adbc.h --silent
-curl -L https://github.com/apache/arrow-adbc/raw/main/c/driver_manager/adbc_driver_manager.h \
-  --output src/adbc_driver_manager.h --silent
+curl -L https://github.com/WillAyd/arrow-adbc/raw/fix-adbc-include/c/include/arrow-adbc/adbc.h \
```

Comment:
This is not meant to be a permanent change, though I think required to get CI green?

Reply:
@paleolimbot thanks for all of the help here. Do you think this change is still required? Not sure how to best manage this script when the location of these files change relative to main

Reply:
Should I change it back to the adbc.h path that is in main today (adbc.h)  or what it will be moved to as part of this PR (c/include/arrow-adbc/adbc.h?